### DISCUSSION TITLE
能否在类组件中使用colorBgContainer, borderRadiusLG

### DISCUSSION THREAD

**Discussion by zw-Ch** [ant-design/ant-design#47082] [Q&A]
在函数组件function中，可以通过以下代码获取colorBgContainer, borderRadiusLG两个属性
`const {token: {colorBgContainer, borderRadiusLG},} = theme.useToken();`
但现在因为部分原因，我不得不使用类组件，
请问可以在类组件中使用colorBgContainer, borderRadiusLG两个属性吗？

**Comment by linxianxi** [comment]
外面包一个函数组件，当 props 传到类组件里即可。
